framework/src/st_things/things_stack/things_types.h

```diff
@@ -83,8 +77,8 @@ typedef enum {
 } wifi_enc_type_e;
 
 typedef struct {
-	char ssid[MAX_SSID_LEN];	/**< ssid of the Enroller**/
-	char pwd[MAX_SECUIRTYKEY_LEN]; /**< pwd of the Enroller**/
+	char ssid[WIFIMGR_SSID_LEN];	/**< ssid of the Enroller**/
```

Comment:
i think you need to len +1 for last null.

Reply:
@harish-191 , Add +1 for NULL character.

Reply:
fixed.